ci/deployment.yaml

```diff
@@ -66,6 +66,9 @@ spec:
            - mountPath: /user-tokens
              name: ci-tokens
              readOnly: true
+           - mountPath: /secrets/db-config
+             name: database-server-config
+             readOnly: true
```

Comment:
CI doesn't need the database-server-config.  Just the pod that's launched by the CI for the create database step.  This means the database secret needs to be in batch-pods.  (This is of course wrong and a long outstanding issue with the current CI/batch design.  I think batch2 makes this much easier to fix, actually.  We should revisit this when switching CI over to batch2.)

Reply:
The seconds on CI deployment are still here.